localization/zh/trampoline/README.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Strategy
```

Comment:
This must be a mistake. Should be Trampoline, right?

Reply:
Yes, thank you for pointing that out. I have fixed it.